examples/echo/main.go

```diff
@@ -11,14 +11,18 @@ import (
 	"log"
 	mrand "math/rand"
 
+	"github.com/golang/glog"
```

Comment:
Oh, sorry. Today I felt like whole day I was on autopilot so I saw a PR and reviewed it.